### ISSUE TITLE
Cannot create GF(65537^12)

### ISSUE THREAD

**Issue by remyoudompheng** [sagemath/sage#35357]
### Is there an existing issue for this?

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.

### Did you read the documentation and troubleshoot guide?

- [X] I have read the documentation and troubleshoot guide

### Environment

```markdown
- **OS**: Archlinux
- **Sage Version**: 9.8
```

### Steps To Reproduce

Try to create the finite field of order 65537^12 (without variable name)
```
sage: GF(65537^12)
```
Or any other exponent which is a multiple of 12.

### Expected Behavior

Create the finite field of order 65537^12

### Actual Behavior

Throws an exception:
```
  File "sage/rings/algebraic_closure_finite_field.py", line 1119, in _get_polynomial
    return self._pseudo_conway_lattice.polynomial(n)
  File "sage/rings/finite_rings/conway_polynomials.py", line 233, in polynomial
    _frobenius_shift(K, xi)
  File "sage/rings/finite_rings/conway_polynomials.py", line 432, in _frobenius_shift
    crt[(i,j)].append(_find_pow_of_frobenius(p, m, compatible[m][qlist[j]], compatible[m][qlist[i]]))
  File "sage/rings/finite_rings/conway_polynomials.py", line 318, in _find_pow_of_frobenius
    raise RuntimeError("No appropriate power of Frobenius found")
RuntimeError: No appropriate power of Frobenius found
```

### Additional Information

This was found randomly while trying to create a doctest.
It seems related to specific arithmetic properties and difficult to reproduce randomly with other values of p and the degree.

**Comment by grhkm21** [comment]
Not-so-minimal reproducible example:

```python
sage: p = 65537
....: Fpbar = GF(p).algebraic_closure("z")
....: try: Fpbar._pseudo_conway_lattice.polynomial(12)
....: except RuntimeError: print("Error")
Error
sage: p = 65537
....: Fpbar = GF(p).algebraic_closure("z", use_database=False)
....: Fpbar._pseudo_conway_lattice.polynomial(12)
x^12 + 49224*x^11 + 29157*x^10 + 43350*x^9 + 35738*x^8 + 17809*x^7 + 41614*x^6 + 28700*x^5 + 11177*x^4 + 5221*x^3 + 38906*x^2 + 31504*x + 3
```

(The Fpbar line is called in `src/sage/rings/finite_rings/finite_field_constructor.py:614`) Not sure if the database is messed up or what's happening, I do not have the necessary knowledge to understand the math.

**Comment by grhkm21** [comment]
Further testing:

```python
sage: from sage.rings.finite_rings.conway_polynomials import *
sage: PCL = PseudoConwayLattice(60013, use_database=True)
....: print(PCL.nodes)
{1: x + 60007, 2: x^2 + 60011*x + 6, 3: x^3 + 7*x + 60007, 4: x^4 + 4*x^2 + 52216*x + 6}
....: PCL.nodes = {4: PCL.nodes[4]}
....: print(PCL.nodes)
....: PCL.polynomial(12) # Error
```

Looking at the [source](http://www.math.rwth-aachen.de/~Frank.Luebeck/data/ConwayPol/CPgt60000.html) of the database, it seems that the Conway polynomials $f_{p, 4}$ are computed by a different program (JB) than that used for $f_{p, k}, k \leq 3$ (GAP). I wonder if the program is just wrong? Or if it's a correct output that is "incompatible" with the rest of the algorithm.

**Comment by grhkm21** [comment]
It seems that this is magically fixed in Version 10.4 now, probably due to #37443.

**Comment by S17A05** [comment]
> Looking at the [source](http://www.math.rwth-aachen.de/~Frank.Luebeck/data/ConwayPol/CPgt60000.html) of the database, it seems that the Conway polynomials fp,4 are computed by a different program (JB) than that used for fp,k,k≤3 (GAP). I wonder if the program is just wrong? Or if it's a correct output that is "incompatible" with the rest of the algorithm.

It seems that the database is indeed wrong, at least if I understood the definitions correctly: The Conway polynomial $f_{p,4}$ for $p = 65537$ given by the database is not compatible with the polynomial $f_{p,2}$ (for which everything seems to agree) since the roots of $$f_{p,4} = x^4 + 7x^2 + 48035x + 3$$ raised to the power $r = (p^4 - 1)/(p^2 - 1) = 4295098370$ are not roots of $$f_{p,2} = x^2 + 46810*x + 3.$$

In contrast, the polynomial computed without the database is $$g = x^4 + 29636 x^3 + 18763 x^2 + 53114 x + 3,$$
and the roots of this polynomial raised to the $r$-th power are indeed roots of $f_{p,2}.$

**Comment by S17A05** [comment]
Fixed in 10.4beta8 via the update of the Conway polynomials database, which was integrated in #38110.
